lib/refile/attachment.rb

```diff
@@ -85,8 +85,8 @@ def attachment(name, cache: :cache, store: :store, raise_errors: true, type: nil
         define_method "remote_#{name}_url" do
         end
 
-        define_method "#{name}_url" do |*args|
-          Refile.attachment_url(self, name, *args)
+        define_method "#{name}_url" do |**args|
+          Refile.attachment_url(self, name, **args)
```

Comment:
the `Refile.attachment_url` accept both `*args` and some other optional keyword arguments as well like `expires_at` etc
https://github.com/refile/refile/blob/c4ac577c6fdad92bc079a62a0e82888319daedc8/lib/refile.rb#L400

```suggestion
        define_method "#{name}_url" do |*args, **kwargs|
          Refile.attachment_url(self, name, *args, **kwargs)
```

Reply:
thanks, I've done single PR with all my latest update for Refile https://github.com/refile/refile/pull/612